pkg/agent/route/interfaces.go

```diff
@@ -87,4 +87,7 @@ type Interface interface {
 
 	// DeleteRouteForLink deletes a route entry for a specific link.
 	DeleteRouteForLink(dstCIDR *net.IPNet, linkIndex int) error
+
+	// ClearConntrackEntryForService deletes a conntrack entry for a Service connection.
```

Comment:
This deletes all entries matching the provided IP/port, not only one, right?

Reply:
Yes, all matching items will be deleted.